.github/workflows/ci.yml

```diff
@@ -371,21 +371,19 @@ jobs:
       - name: "Validate global Python install"
         run: python scripts/check_system_python.py --uv ./uv
 
+  # Note: rockylinux:8 is a 1-1 code compatible distro to rhel 8
+  # rockylinux:8 mimics centos:8 but with added maintenance stability
+  # and avoids issues with centos stream mirror issues
   system-test-centos:
-    # https://github.com/astral-sh/uv/issues/2915
-    if: false
     needs: build-binary-linux
-    name: "check system | python on centos"
+    name: "check system | python on centos (via rockylinux)"
```

Comment:
Should this just be `python on rockylinux`? And should the job be `system-test-rockylinux`? Or would it be standard to still refer to this as CentOS? I'm not really familiar.

Reply:
Makes sense, renamed in dd7b8952752f3ee8da2e8159233ec191c477e35e

Ultimately, testing on Rocky Linux achieves the same effect of testing compatibility with CentOS and RHEL. Generally speaking, RHEL is not gonna make another CentOS release either so this gives us the best of both worlds right now given the state of affairs with RedHat.